.github/workflows/ci.yml

```diff
@@ -38,6 +51,16 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ env.NODE_VERSION }}
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: | 
+            node_modules
```

Comment:
should we try to install root package in case there is a difference between the node modules in develop vs the PR?
(For example the PR introduces an additional library or library versions differ)

Reply:
for this particular job, no because unit tests job won't test e2e.

Moreover, the e2e job(s), install packages in both places from the script, so no need as well